### PR TITLE
Fix #224 move format out of synchronized block

### LINKED ISSUE #224: All request in JSF are synchronized in the PrettyTimeConverter
Because `PrettyTime.format()` is called inside an synchronized block.

### PR DESCRIPTION
**Pull request by ivy-cst**

### CODE REVIEW

**Review by gastaldi — APPROVED** [review]

**Comment by ivy-cst** [comment]
@gastaldi Thanks for approving. Could you please also merge? I do not have write access and can not merge.

**Comment by gastaldi** [comment]
@ivy-cst me neither, it's up to @lincolnthree  to do this 😉

**Comment by lincolnthree** [comment]
Thanks @gastaldi and @ivy-cst ! Much appreciated. Cutting a new version now :)

**Pull request merged**
